### PR TITLE
Remove server local storage from code

### PR DESCRIPTION
**Pull request by PavelKhripkov**
PMM-0

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

File: agent/runner/jobs/backup_location.go [hunk 1]
```diff
@@ -19,8 +19,8 @@ type BackupLocationType string
 
 // BackupLocation types. Same as in managed/models/location_model.go.
 const (
-	S3BackupLocationType        BackupLocationType = "s3"
-	PMMClientBackupLocationType BackupLocationType = "pmm-client"
+	S3BackupLocationType         BackupLocationType = "s3"
+	FilesystemBackupLocationType BackupLocationType = "pmm-client"
```
Comment (artemgavrilov) [inline agent/runner/jobs/backup_location.go:23]: It seems that we can also rename value here and in `managed/models/location_model.go`

Comment (idoqo) [inline agent/runner/jobs/backup_location.go:23]:
```suggestion
	FilesystemBackupLocationType BackupLocationType = "filesystem"
```

**Review by idoqo — APPROVED** [review]

File: agent/runner/jobs/backup_location.go (hunk 1 above)
Comment (PavelKhripkov) [reply agent/runner/jobs/backup_location.go]: Double) Looks like I really have to change it)))

Comment (PavelKhripkov) [reply agent/runner/jobs/backup_location.go]: Double) Looks like I really have to change it)))

**Pull request merged**